license.Rmd

```diff
@@ -214,11 +185,35 @@ We only add core developers[^license-4] to the `DESCRIPTION` file; but some proj
 [^license-4]: i.e. people responsible for on-going development.
     This is best made explicit in the ggplot2 governance document, [`GOVERNANCE.md`](https://github.com/tidyverse/ggplot2/blob/main/GOVERNANCE.md).
 
+### Relicensing
+
+It's important to think about your initial license because it's hard to change it later because it requires the permission of all copyright holders.
+Unless you have a CLA, this will include everyone who has contributed a non-trivial amount of code.
```

Comment:
```suggestion
Unless you have a so-called contributor license agreement (CLA), this will include everyone who has contributed a non-trivial amount of code.
```

Reply:
CLA is now defined in the paragraph above.